src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -27,6 +29,20 @@
           (style/create-link #(on-back)
             (icons/font-icon {:style {:fontSize "70%" :marginRight "0.5em"}} :angle-left)
             "Back to table")
+          [:div {:style {:marginTop "1em"}}
+           (style/create-section-header (str "Selected Configuration : "
```

Comment:
The more common convention in our codebase is "Thing: foo" (no space after the first word). Let's keep to that.